src/DatabaseLibrary/assertion.py

```diff
@@ -20,7 +20,7 @@ class Assertion(object):
     Assertion handles all the assertions of Database Library.
     """
 
-    def check_if_exists_in_database(self, selectStatement, sansTran=False):
+    def check_if_exists_in_database(self,alias, selectStatement, sansTran=False):
```

Comment:
enable some PEP8 plugin to remark missing or double spaces.
```suggestion
    def check_if_exists_in_database(self, alias, selectStatement, sansTran=False):
```